drivers/dai/intel/hda/Kconfig.hda

```
@@ -0,0 +1,13 @@
+# SOF HDA configuration options
+
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_INTEL_HDA_DAI := intel,hda-dai
+
+config DAI_INTEL_HDA
+	bool "Intel HDA driver for Dai interface"
+	default $(dt_compat_enabled,$(DT_COMPAT_INTEL_HDA_DAI))
```

Use the `DT_HAS_<compat>_ENABLED` helpers.

```suggestion
	default y
	depends on DT_HAS_INTEL_HDA_DAI_ENABLED
```